src/SFA.DAS.EmployerIncentives.Api/Authorization/LocalAuthorizationHandler.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+
+namespace SFA.DAS.EmployerIncentives.Api.Authorization
+{
+    public class LocalAuthorizationHandler : AuthorizationHandler<NoneRequirement>
+    {
+        protected override Task HandleRequirementAsync(AuthorizationHandlerContext context,
```

Comment:
As per previous, looks like this could be shared?

Reply:
yes